packages/sdk-react/src/components/providers/FusionAuthProvider.test.tsx

```diff
@@ -250,7 +250,12 @@ describe('FusionAuthProvider', () => {
 
     expect(fetch).toHaveBeenCalledTimes(1);
     expect(onAutoRefreshFailure).toHaveBeenCalledWith(
-      Error(JSON.stringify({ message: 'cannot refresh token' })),
+      Error(
```

Comment:
` Invokes `onRefreshFailure` if an error is thrown during autorefresh` is failing when I run the tests 

```
AssertionError: expected "spy" to be called with arguments: [ …(1) ]

Received: 

  1st spy call:

  Array [
-   [Error: {"status":400,"details":"{\"message\":\"cannot refresh token\"}"}],
+   [Error: {"message":"cannot refresh token"}],
  ]
```

Reply:
@BriDavidson Can you please run `yarn build:core` and try again?

Reply:
@JakeLo123 that worked! thank you

Reply:
Great! Thanks!